packaging/ubuntu-16.04/rules

```diff
@@ -127,7 +127,7 @@ override_dh_auto_build:
 	# Generate static snap-exec - it somehow includes CGO so we must
 	# force a static build here. We need a static snap-exec inside
 	# the core snap because not all bases will have a libc
-	(cd _build/bin && GOPATH=$$(pwd)/.. CGO_ENABLED=0 go build $(DH_GOPKG)/cmd/snap-exec)
+	(cd _build/bin && GOPATH=$$(pwd)/.. CGO_ENABLED=0 go build $(GCCGOFLAGS) $(DH_GOPKG)/cmd/snap-exec)
```

Comment:
doesn't 14.04 need the same thing?

Reply:
Not strictly, it is only needed because the build fails on powerpc. However we could add it on 14.04 to keep the delta smaller. Let me add it.